linechart.py

```diff
@@ -389,8 +495,8 @@ def render(table, params, *, input_columns):
         chart = form.make_chart(table, input_columns)
     except GentleValueError as err:
         return (table, "", {"error": str(err)})
-    except ValueError as err:
-        return (table, str(err), {"error": str(err)})
+    except RenderingError as err:
```

Comment:
Unfortunately, `str(err)` on the line above won't be translatable for the forseeable future. So let's nix it.

Pass `{"error": error: err.args[0]}` -- even though we won't be translating it. Modify `linechart.html`'s `errorSpec()` to set `"title": "Please correct the error in this step's data or parameters"`. In the future, we'll probably have a good way to handle this data.

Reply:
`err.args[0]` is not JSON serializable. I am passing `"Please correct the error in this step's data or parameters"` for now